packages/mobile-app/data/facades/index.ts

```diff
@@ -7,7 +7,7 @@ import { chainHandlers } from "./chain/handlers";
 import { chainDemoHandlers } from "./chain/demoHandlers";
 import { contactsDemoHandlers } from "./contacts/demoHandlers";
 
-const DEMO = true;
+const DEMO = false;
```

Comment:
suggestion: Should we make this an environment variable?

Reply:
Ah, let me roll this back. Yeah, if possible I think an environment variable would be good, I'm not sure how those get passed through Metro. Ultimately it'd probably be nice to just run `start-demo` or something similar rather than having to change any files